### PR TITLE
Add refresh button for UWP version of quickstart

### PR DESCRIPTION
**Pull request by bmourat**

### CODE REVIEW

File: client/xamarin.forms/ZUMOAPPNAME/TodoList.xaml.cs [hunk 1]
```diff
@@ -13,17 +13,25 @@ public TodoList()
...
-                syncButton.Clicked += OnSyncItems;
-
-                buttonsPanel.Children.Add(syncButton);
+                refreshButton.Clicked += OnRefreshItems;
+                buttonsPanel.Children.Add(refreshButton);
+                if (manager.IsOfflineEnabled)
+                {
+                    var syncButton = new Button
+                    {
+                        Text = "Sync items",
```
Comment (jwallra) [inline client/xamarin.forms/ZUMOAPPNAME/TodoList.xaml.cs]: Use text "refresh" to align with other quickstarts

File: client/xamarin.forms/ZUMOAPPNAME/TodoList.xaml.cs [hunk 2]
```diff
@@ -13,17 +13,25 @@ public TodoList()
             InitializeComponent();
 
             manager = TodoItemManager.DefaultManager;
-			
-            if (manager.IsOfflineEnabled && Device.RuntimePlatform == Device.UWP)
+            if (Device.RuntimePlatform == Device.Windows)
```
Comment (dhei) [inline client/xamarin.forms/ZUMOAPPNAME/TodoList.xaml.cs]: Revert to `Device.UWP`

**Comment by jwallra** [comment]
I'll test it out now and approve when finished if everything looks good

File: client/xamarin.forms/ZUMOAPPNAME/TodoList.xaml.cs (hunk 1 above)
Comment (dhei) [reply client/xamarin.forms/ZUMOAPPNAME/TodoList.xaml.cs]: @jwallra  "Refresh" text is used at https://github.com/Azure/azure-mobile-apps-quickstarts/pull/117/files#diff-f29c1ebd6b7e759cc4ed1438b33a1a17R20. This "Sync items" text is not change at all , see https://github.com/Azure/azure-mobile-apps-quickstarts/pull/117/files#diff-f29c1ebd6b7e759cc4ed1438b33a1a17L21.

**Pull request merged**